lib/fog/compute/kubevirt.rb

```diff
@@ -84,26 +137,162 @@ def initialize(options={})
           # Kubeclient needs different client objects for different API groups. We will keep in this hash the
           # client objects, indexed by API path/version.
           @clients = {}
+        end
+
+        def virt_supported?
+          virt_enabled = false
 
-          @client = kubevirt_client()
+          begin
+            api_versions = kubevirt_client.api["versions"]
```

Comment:
I think we can refactor:
def virt_supported?
   kubevirt_client.api["versions"].any? { |ver| ver["groupVersion"]&.start_with?(KUBEVIRT_GROUP) }
rescue => err
   ....
end

Reply:
Will check